content.js

```diff
@@ -57,14 +57,17 @@ function hidePopup() {
 
 function findCard() {
     // Select a random a card in between a range
-    let cardPositionIndex = 1
+    let cardPositionIndex = 0
```

Comment:
I ignored the `ad-renderer` videos in `querySelector` below so that we can consistently start at index 0 on the homepage and the video page (suggestions)